examples/todo.rs

```diff
@@ -30,16 +31,23 @@ use rustful::StatusCode;
 
 //Helper for setting a status code and then returning from a function
 macro_rules! or_abort {
-    ($e: expr, $response: expr, $status: expr) => (
+    ($e: expr, $response: ident, $error_message: ident) => (
```

Comment:
yeah, you are definitely right. Don't know why I decided to be strict as possible there :)